omero/sysadmins/cli/admin.txt

```diff
@@ -6,22 +6,95 @@ Server start
 
 Once your database has been properly configured and your config profile
 is set to use that database, you are ready to start your server using the
-:omerocmd:`admin start` command::
+:program:`omero admin start` command::
 
     $ bin/omero admin start
 
-.. note::
+This command performs the following operations in order:
 
-    :omerocmd:`admin start` and :omerocmd:`admin restart` provide a useful
-    debugging and maintenance option called :option:`--foreground`. Using
-    this option allows for starting the server up in the foreground, that is
+#. rewrites the configuration files if :option:`--force-rewrite` is passed or
+   the server has never been started
+#. checks the server status, i.e. pings the ``master`` node using the IceGrid
+   administration tool
+#. aborts the command if a server is running
+#. rewrites the configuration files if it has not been done at step 1
+#. starts the server
+#. waits until the server is up
+
+Most configuration files under :file:`etc/grid` are generated using the
+templates under :file:`etc/grid/templates` and the server configuration stored
+in :file:`etc/grid/config.xml`. During this rewriting step, the JVM memory
+settings are recalculated (see :ref:`jvm_memory_settings`) and the new server
+ports (see :ref:`ports_configuration`).
```

Comment:
It feels like we're left waiting for the verb re. what happens to the new server ports ("ports" is noun, not verb, right?). Should this be, "during Foo the Bar and Baz are recalculated"?